modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -15,7 +15,7 @@ The cluster administrator can use this performance profile configuration to make
 
 [NOTE]
 ====
-The RT kernel is only supported on worker nodes.
+The RT kernel is supported on worker nodes when you use a standard 3+3 cluster, but there are exceptions for compact and single node OpenShift (SNO) deployments. In SNO installations, which contain only a master node, the RT kernel is supported on the master.
```

Comment:
**RT kernel**: abbreviation not defined. Considering the usage above, I'd just call this kernel-rt.
**single node OpenShift (SNO) deployments**: We don't use this name. Instead, we need to describe it generically, like "installing OpenShift on a single node". It's a long story.
**a standard 3+3 cluster**: I've never encountered this standard. In the context, I wonder whether we can simply distinguish between single-node deployments and everything else.

Consider something like this:

In most deployments, kernel-rt is supported only on worker nodes. For installations of <product-title> on a single node, kernel-rt is supported on the single control-plane node.

Reply:
- Changed RT kernel to kernel-rt.
- Yeah, I'm quite aware of the SNO controversy. I'll reword to avoid it.
- Rewording the 3+3 sentence.